fedora/Dockerfile

```diff
@@ -1,13 +1,9 @@
-FROM fedora:20
+FROM fedora
 MAINTAINER amitsaha.in@gmail.com
 
-# Let's start with some basic stuff.
-RUN yum -y clean all
-RUN yum -y update
-RUN yum install -y iptables ca-certificates lxc e2fsprogs
-
-# Install Docker from Fedora repos
-RUN yum -y install docker-io
+# Update and install docker
+RUN dnf -y update && dnf clean all
+RUN dnf -y install docker && dnf clean all
```

Comment:
I left the RUN statements separate because update and install can be costly and Fedora mirrors break all the time for me, but if you feel strongly they certainly can be combined.

You are correct that yum is an alias, with the caveat that an ugly red warning appears every time it is invoked instead of dnf.

I'll leave it to you whether you'd prefer to source the install from docker.